scripts/synthetix/facilitate_trade.js

```diff
@@ -79,21 +68,16 @@ module.exports = async (callback) => {
     console.log("Using account", account)
 
     const snxjs = new SynthetixJs({ networkId: networkId })
-    const exchange = await getExchange(web3)
-
-    const sETH = sETHByNetwork[networkId]
-    const sUSD = sUSDByNetwork[networkId]
+    const batchExchange = await getExchange(web3)
```

Comment:
Not introduced in this PR, but I just noticed:
```suggestion
    const batchExchange = await getExchange()
```